lib/composables/Utils.ts

```diff
@@ -61,3 +62,11 @@ export function useSlotValue(name = 'default'): ComputedRef<string | null> {
     return v !== '' ? v : null
   })
 }
+
+export function tryOnMounted(fn: () => void): void {
```

Comment:
Not using vueuse's version because it executes the function in SSR which is not expected. Also this versions aligns better with vue as vue too doesn't run onMounted's callback in SSR.